gensim/similarities/docsim.py

```diff
@@ -1013,7 +1013,7 @@ class WmdSimilarity(interfaces.SimilarityABC):
         >>> from gensim.models import Word2Vec
         >>> from gensim.similarities import WmdSimilarity
         >>>
-        >>> model = Word2Vec(common_texts, vector_size=20, min_count=1)  # train word-vectors
+        >>> model = Word2Vec(common_texts, vector_size=20, min_count=1).wv  # train word-vectors
         >>>
         >>> index = WmdSimilarity(common_texts, model)
```

Comment:
Clearer to fetch the KeyedVectors from the trained model here, than above?

```suggestion
        >>> index = WmdSimilarity(common_texts, model.wv)
```